### PR TITLE
Zen-browser: Fix Bookmark search

### LINKED ISSUE #15129: [Zen Browser] Bookmarks / Profile Selection Issue
### Extension

https://www.raycast.com/Keyruu/zen-browser

### Raycast Version

1.84.3

### macOS Version

15.0.1

### Description

The only bookmarks shown to me in "Search Bookmarks" are the 5 default bookmarks. I'm not sure where they're coming from. I have one profile, and one profile directory. It has these default bookmarks deleted and several others created. The history in "Search History" and
"New Tab" are correct, though.

Screenshot 2024-10-28 at 11 34 24

Screenshot 2024-10-28 at 11 29 28

Screenshot 2024-10-28 at 11 30 45

### Steps To Reproduce

I'm really not sure how to reproduce this one, but I've provided as much information as I have.

### Current Behaviour

I get the default bookmarks from a profile that doesn't exist. Maybe a cache thing.

### Expected Behaviour

I can search my bookmarks.

### PR DESCRIPTION
**Pull request by Keyruu**
## Description

I fixed #15129. Instead of using the bookmarkbackups directory, it using the SQlite DB.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/zen-browser/package.json [hunk 1]
```diff
@@ -5,11 +5,7 @@
   "description": "Search and open tabs from bookmarks and history in Zen Browser.",
   "icon": "zen-logo.png",
   "author": "Keyruu",
-  "contributors": [
```
Comment (pernielsentikaer) [inline extensions/zen-browser/package.json:8]: Why did you remove all contributors?

**Review by theherk — APPROVED** [review]
I have run from this branch and verified that this works correctly. The bookmarks are now updated simply by restarting the browser, no longer needing to backup bookmarks. Thanks @Keyruu.

**Comment by pernielsentikaer** [comment]
@Keyruu could you check my question?

File: extensions/zen-browser/package.json (hunk 1 above)
Comment (Keyruu) [reply extensions/zen-browser/package.json:8]: Because they get pinged for every issue and they didn't contributed to this extension only for the Firefox extension. I initially included them because I copied a bunch of code from that extension.
